syndesis/plugins/org.fusesource.ide.syndesis.extension.ui/src/org/fusesource/ide/syndesis/extensions/ui/internal/Trace.java

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+ * Copyright (c) 2003, 2011 IBM Corporation and others.
```

Comment:
2017

Reply:
its not our copyright, so I don't adapt the dates